apps/web/src/editor.css

```
@@ -33,6 +33,95 @@
+.link-reader-shell {
+  --link-reader-background: var(--background);
+  --link-reader-background-alt: color-mix(
+    in oklch,
+    var(--foreground) 4%,
+    var(--background)
+  );
+  --link-reader-border: color-mix(in oklch, var(--foreground) 10%, transparent);
+  --link-reader-text: var(--foreground);
+  --link-reader-muted: var(--muted-foreground);
+  --link-reader-accent: var(--foreground);
+  position: relative;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)